plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -424,7 +422,7 @@ public String getPluginVersion(File file) {
                 VersionNumber pluginVersion = new VersionNumber(getPluginVersion(file));
                 installedPluginVersions.put(pluginName, pluginVersion);
                 installedPlugins.add(pluginName);
-		System.out.println(pluginName);
+                System.out.println(pluginName);
```

Comment:
Might want to add this to logging.  You can build the list of downloaded plugins in the same way as the log statement for the failed plugins

Reply:
Added in 0c27fbb.